output/csvwriter.go

```diff
@@ -1 +1,68 @@
 package output
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"reflect"
+	"strings"
+)
+
+type CsvWriter struct {
+	file *os.File
+}
+
+func (c *CsvWriter) Init(filename string) error {
+	file, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	c.file = file
+	err = writeHeaders(c.file)
```

Comment:
WriteHeaders should be accessed from outside, it is part of interface

Reply:
I think it is just csv implementation specific. What would this method actually do in DB implementation? I also don't think it should be public: we only call it when initializing file. If we had some weird implementation that e.g. breaks the output into multiple files I would still argue, that it would be better to deal with it internally in the writer and let the client just say "hey, here's some data, save it for me". What do you think?

Reply:
ok, I was thinking about order of columns in database, but we already have struct, so it is not necessary, we should remove it from the interface also